### PR TITLE
Update sidebar. Add sidepage search. Update typography

### PR DESCRIPTION
**Pull request by AtkishkinVlad**

### CODE REVIEW

File: components/CommonLinks/CommonLinks.module.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.CommonLinks__container {
+    position: fixed;
+    display: flex;
+    align-items: center;
+    background-color: rgb(239 240 241);
+    border-radius: 64px;
+    width: calc(100vw);
```
Comment (a-chabin) [inline components/CommonLinks/CommonLinks.module.css:7]: `calc(100vw)` -> `100vw`

File: components/File/File.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import Image from 'next/image'
+import React from 'react'
+
+function File({ columnItem }) {
+    const src = columnItem?.content?.external?.url
+
+    return (
+        <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'flex-start' }}>
```
Comment (a-chabin) [inline components/File/File.jsx:8]: Move styles to `File.module.css`

File: components/ManualPage/ManualPage.module.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    text-align: center;
+    margin-top: 1vw;
+}
+
+br + .Manual__image__container {
+	margin-top: 3vw;
+}
+
+.Manual__image__container > img {
+    position: unset !important;
```
Comment (a-chabin) [inline components/ManualPage/ManualPage.module.css:53]: Let's use `static`

File: components/File/File.jsx (hunk 2 above)
Comment (AtkishkinVlad) [reply components/File/File.jsx]: Fixed

File: components/CommonLinks/CommonLinks.module.css (hunk 1 above)
Comment (AtkishkinVlad) [reply components/CommonLinks/CommonLinks.module.css]: Fixed

File: components/ManualPage/ManualPage.module.css (hunk 3 above)
Comment (AtkishkinVlad) [reply components/ManualPage/ManualPage.module.css]: Fixed

**Review by a-chabin — APPROVED** [review]

**Pull request merged**